SPECS/libmspack/libmspack.spec

```diff
@@ -1,20 +1,21 @@
 Summary:        A library that provides compression and decompression of file formats used by Microsoft
 Name:           libmspack
-Version:        0.7.1alpha
-Release:        4%{?dist}
+Version:        0.10.1alpha
+Release:        1%{?dist}
 License:        LGPLv2+
-URL:            http://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
-Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
```

Comment:
Can we change the URL to not reference the old tarball?

Reply:
Fixed